### ISSUE TITLE
Downloaded scss files override current project's

### ISSUE THREAD

**Issue by MrBrax** [Facepunch/sbox-issues#5292]
### Describe the bug

A scss file in `.source2/cloud/` will override the one you're working on in your local project regardless of what package it belongs to if it has the same path and name. I think it overrides razor files too.

### To Reproduce

1. Download package that has scss file uploaded
2. Use the same path and filename in local project
3. Local file won't be used

### Expected behavior

Cloud files should not override anything.

### Media/Files

![1711617169187](https://github.com/Facepunch/sbox-issues/assets/1517911/c16a38cd-d76a-4d2c-96f1-472eaf221bc9)

![1711616467471](https://github.com/Facepunch/sbox-issues/assets/1517911/fa7a4a7b-0584-4ba6-9dd4-76a866e9d4ef)

i don't have a menu.razor.scss file, when i remove it from cloud, i get this error:

![1711617479330](https://github.com/Facepunch/sbox-issues/assets/1517911/35fe3087-29a1-4a68-aca9-f1ece0f483d4)

**Comment by MrBrax** [comment]
i don't believe this is happening anymore

**Comment by MrBrax** [comment]
this is apparently happening to prefabs and scene files

**Comment by garrynewman** [comment]
Should be better now?

**Comment by FlooferLand** [comment]
I've had this issue as well but I haven't been able to reproduce it.
Made a thread at #6408 discussing the issue more thoroughly as I tested it before eventually fixing it.

Removing `sbox/.source2/`, `sbox/download/`, and recompiling assets solved it on my end.
I think I triggered the bug by dragging assets from one game into another using the OS file manager, and it might require the game the assets originated from having been previously built.
Again, guesses
